### ISSUE TITLE
[Kotlindoc] Default output directory is incorrect for multi-project builds

### ISSUE THREAD

**Issue by tlinkowski** [aalmiray/kordamp-gradle-plugins#161]
When `kotlindoc.outputDirectory` is not explicitly set for every subproject, it's implicitly set to a root-project-relative directory.

This happens during the merge of subproject's config with the root project's config:
https://github.com/aalmiray/kordamp-gradle-plugins/blob/39c56aec385bc51b3060940b51fee3ecf4bdc324/plugins/base-gradle-plugin/src/main/groovy/org/kordamp/gradle/plugin/base/BasePlugin.groovy#L346-L349

where:

- `rootExtension.kotlindoc.outputDirectory = [root-project]\build\docs\kotlindoc`, which is a result of:
https://github.com/aalmiray/kordamp-gradle-plugins/blob/39c56aec385bc51b3060940b51fee3ecf4bdc324/plugins/base-gradle-plugin/src/main/groovy/org/kordamp/gradle/plugin/base/BasePlugin.groovy#L341-L343
and
https://github.com/aalmiray/kordamp-gradle-plugins/blob/d147b32200b5624b9e1bed0edd7ea46e1a285823/plugins/base-gradle-plugin/src/main/groovy/org/kordamp/gradle/plugin/base/plugins/Kotlindoc.groovy#L285-L286

- `extension.kotlindoc.outputDirectory = null`

To sum up:
- actual behavior: every subproject gets `extension.kotlindoc.outputDirectory = [root-project]\build\docs\kotlindoc`.
- expected behavior: every subproject gets `extension.kotlindoc.outputDirectory = [subproject]\build\docs\kotlindoc`.

Affected version: 0.25.0.
